Arrays-Implementation/subarray/subsetOfArray.cpp

```diff
@@ -0,0 +1,23 @@
+//Given two arrays: a1[0..n-1] of size n and a2[0..m-1] of size m. Task is to check whether a2[] is a subset of a1[] or not.
+//Both the arrays can be sorted or unsorted. It may be assumed that elements in both array are distinct.
```

Comment:
Please add the required "include" statements to make the required program to actually work